policies/genai.md

```diff
@@ -112,3 +117,10 @@ non-AI tool. For example, tools like [Dosu](https://dosu.dev/) are being used in
 certain repositories to aid in code review and issue management. Remember that
 these tools can make mistakes, and use your best judgement when evaluating their
 output.
+
+_Q: Can I ask a contributor to disclose if LLM or Generative AI tools were used to 
+create a contribution?_
+
+Yes. Authors should voluntarily offer this information, but it could have been
```

Comment:
I'm not sure about this statement. When I send a PR, it's not obvious to me that I should proactively state that I used GenAI tools.

If maintainers want to know this information, I'd suggest adding it to the PR template, e.g. https://github.com/renovatebot/renovate/blob/main/.github/pull_request_template.md#ai-assistance-disclosure